db/add_missing_instructor_emails.sql

```diff
@@ -0,0 +1,9 @@
+-- Copy `email`s from CourseInstructorGrade to Instructor
+-- assuming instructor names are unique

```

Comment:
We make this assumption in our models and migration so I think it's ok but this might be the source of quirks in our data. Also, if there are duplicates would it just replace with the first email it finds under the name?

Reply:
Just verified that 1292 `Instructor` instances are updated in both cases (when you run all the commands in this PR), so I took out the unnecessary assumption in cb31ecc5c58c5de5da9715e6019dbb1a00e221f6.